packages/bar/src/Bar.tsx

```diff
@@ -362,6 +364,19 @@ const InnerBar = <RawDatum extends BarDatum>({
         )
     }
 
+    if (layers.includes('totals')) {
```

Comment:
Usually, I tend to expose a prop rather than having to deal with layers directly, something like `enableTotals` with the layer enabled by default, I feel it's a bit simpler to use than having to redefine all layers, what do you think?

Reply:
It also makes it easier to document, right now, appart from the story, it'll be hard to tell that this feature exists.

Reply:
done, this way is much better, thank you!

I made the default behavior as not enabled in order to don't affect production charts. What do you think?